### DISCUSSION TITLE
Dropdown not working properly

### DISCUSSION THREAD

**Discussion by majadul1989** [pratikborsadiya/vali-admin#137] [Q&A]
Drop down not working properly it by double click than it show dropdown

**Comment by majadul1989** [comment]
![Screenshot](https://user-images.githubusercontent.com/24989221/90216054-56cf4c80-de1f-11ea-8b64-a67fd3d3d0a9.png)

**Comment by pratikborsadiya** [comment]
Vue JS support is not native. Converting to a discussion if someone with Vue knowledge wants to help.
